rocAL/rocAL_pybind/amd/rocal/fn.py

```diff
@@ -29,219 +29,437 @@
+    kernel_size = b.createIntParameter(kernel_size) if isinstance(kernel_size, int) else kernel_size
     # pybind call arguments
-    kwargs_pybind = {"input_image0": inputs[0],"is_output": False, "sdev": None}
-    blur_image = b.Blur(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+    kwargs_pybind = {"input_image": inputs[0], "is_output": False, "kernel_size": kernel_size,
+                     "rocal_tensor_output_layout": rocal_tensor_output_layout, "rocal_tensor_output_datatype": rocal_tensor_output_datatype}
+    blur_image = b.Blur(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     return (blur_image)
 
-def contrast(*inputs, bytes_per_sample_hint=0, contrast=1.0, image_type=0, min_contrast=None, max_contrast=None,
-             preserve=False, seed=-1, device=None):
+
+def contrast(*inputs, contrast_factor=None, contrast_center=None, rocal_tensor_output_layout=types.NHWC, rocal_tensor_output_datatype=types.UINT8):
     """
-    bytes_per_sample_hint (int, optional, default = 0) – Output size hint (bytes), per sample. The memory will be preallocated if it uses GPU or page-locked memory
+    inputs - the input image passed to the augmentation
+    
+    contrast_factor (float, optional, default = None) - contrast multiplier used for the augmentation. Values >= 0 are accepted. For example: 0 - gray image, 1 - no change, 2 - increase contrast twice
 
-    contrast (float, optional, default = 1.0) –
+    contrast_center (float, optional, default = None) - intensity value unaffected by the augmentation
 
-    Contrast change factor. Values >= 0 are accepted. For example:
+    rocal_tensor_output_layout (int, optional, default = types.NHWC) - tensor layout for the augmentation output
 
-    0 - gray image,
+    rocal_tensor_output_datatype (int, optional, default = types.UINT8) - tensor dtype for the augmentation output
+    """
+    contrast_factor = b.createFloatParameter(contrast_factor) if isinstance(contrast_factor, float) else contrast_factor
+    contrast_center = b.createFloatParameter(contrast_center) if isinstance(contrast_center, float) else contrast_center
 
-    1 - no change
+    # pybind call arguments
+    kwargs_pybind = {"input_image": inputs[0],
+                     "is_output": False, "contrast_factor": contrast_factor, "contrast_center": contrast_center, "rocal_tensor_output_layout": rocal_tensor_output_layout, "rocal_tensor_output_datatype": rocal_tensor_output_datatype}
+    contrast_image = b.Contrast(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
+    return (contrast_image)
 
-    2 - increase contrast twice
 
-    image_type (int, optional, default = 0) – The color space of input and output image
+def flip(*inputs, h_flip=0, v_flip=0, rocal_tensor_output_layout=types.NHWC, rocal_tensor_output_datatype=types.UINT8):
+    """
+    inputs - the input image passed to the augmentation
+    
+    h_flip (int, optional, default = None) - flip the horizontal dimensions
```

Comment:
We have set 0 as default but you are stating default is None. Please check.

Reply:
Changed the default value in the docstring